lib/extension/list.extension.js

```diff
@@ -0,0 +1,40 @@
+'use strict'; // eslint-disable-line strict
+/**
+ *  Class of an extension doing the simple listing
+ */
+class List {
+    /**
+     *  Constructor
+     *  Set the logger and the res
+     *  @param {Object} parameters - The parameters you sent to DBD
+     *  @param {RequestLogger} logger - The logger of the request
+     *  @return {undefined}
+     */
+    constructor(parameters, logger) {
```

Comment:
Yep, the extension have to follow a constructor that takes parameters and logger.